library/std/src/sys/unix/ext/fs.rs

```diff
@@ -345,6 +345,31 @@ pub trait OpenOptionsExt {
     /// ```
     #[stable(feature = "open_options_ext", since = "1.10.0")]
     fn custom_flags(&mut self, flags: i32) -> &mut Self;
+
+    /// Get the flags of this OpenOptions as [`libc::c_int`].
```

Comment:
```suggestion
    /// Get the flags as [`libc::c_int`].
```
`of this OpenOptions` seemed unnecessary